packages/actor-optimize-query-operation-filter-pushdown/lib/ActorOptimizeQueryOperationFilterPushdown.ts

```diff
@@ -161,6 +174,25 @@ export class ActorOptimizeQueryOperationFilterPushdown extends ActorOptimizeQuer
       return true;
     }
 
+    // Don't push down extension functions comunica support, but an endpoint doesn't
+    if (comunicaFunctions && expression.expressionType === Algebra.expressionTypes.NAMED) {
+      const extensionFunction = expression.name.value;
+      const extensionFunctionExpression: Algebra.NamedExpression = {
```

Comment:
Why do we need to create a new named expression? Can't we just pass the current `expression` directly?